The doc subpkg of trurl fails the doc test

The documentation subpackage of trurl fails testing with the following error:

```
2025/02/04 22:46:58 INFO       --no-guess-scheme            - require scheme in URLs
2025/02/04 22:46:58 INFO       --punycode                   - encode hostnames in punycode
2025/02/04 22:46:58 INFO       --qtrim [what]               - trim the query
2025/02/04 22:46:58 INFO       --query-separator [letter]   - if something else than '&'
2025/02/04 22:46:58 INFO       --quiet                      - Suppress (some) notes and comments
2025/02/04 22:46:58 INFO       --redirect [URL]             - redirect to this
2025/02/04 22:46:58 INFO       --replace [data]             - replaces a query [data]
2025/02/04 22:46:58 INFO       --replace-append [data]      - appends a new query if not found
2025/02/04 22:46:58 INFO   -s, --set [component]=[data]     - set component content
2025/02/04 22:46:58 INFO       --sort-query                 - alpha-sort the query pairs
2025/02/04 22:46:58 INFO       --url [URL]                  - URL to work with
2025/02/04 22:46:58 INFO       --urlencode                  - show components URL encoded
2025/02/04 22:46:58 INFO   -v, --version                    - show version
2025/02/04 22:46:58 INFO       --verify                     - return error on (first) bad URL
2025/02/04 22:46:58 INFO  URL COMPONENTS:
2025/02/04 22:46:58 INFO   url, scheme, user, password, options, host, port, path, query, fragment, zoneid
2025/02/04 22:46:58 WARN + exit 0
2025/02/04 22:46:58 INFO running test pipeline for subpackage trurl-doc
2025/02/04 22:46:58 INFO building test workspace in: '/home/user/tmp/melange-guest-2721097524-trurl-doc' with apko
2025/02/04 22:46:58 [DEBUG] GET https://packages.wolfi.dev/os/apk-configuration
2025/02/04 22:46:58 INFO setting apk repositories: [/home/user/src/wolfi-os/packages https://packages.wolfi.dev/os]
2025/02/04 22:46:58 INFO image configuration:
2025/02/04 22:46:58 INFO   contents:
2025/02/04 22:46:58 INFO     build repositories: []
2025/02/04 22:46:58 INFO     runtime repositories: []
2025/02/04 22:46:58 INFO     keyring:      []
2025/02/04 22:46:58 INFO     packages:     [trurl-doc apk-tools grep man-db texinfo]
2025/02/04 22:46:58 INFO installing ca-certificates-bundle (20241121-r1)
2025/02/04 22:46:58 INFO installing wolfi-baselayout (20230201-r16)
2025/02/04 22:46:58 INFO installing glibc (2.40-r8)
2025/02/04 22:46:58 INFO installing ld-linux (2.40-r8)
2025/02/04 22:46:58 INFO installing libgcc (14.2.0-r8)
2025/02/04 22:46:58 INFO installing glibc-locale-posix (2.40-r8)
2025/02/04 22:46:58 INFO installing zlib (1.3.1-r5)
2025/02/04 22:46:58 INFO installing libcrypto3 (3.4.0-r6)
2025/02/04 22:46:58 INFO installing libssl3 (3.4.0-r6)
2025/02/04 22:46:58 INFO installing apk-tools (2.14.4-r2)
2025/02/04 22:46:58 INFO installing libpcre2-8-0 (10.44-r2)
2025/02/04 22:46:58 INFO installing grep (3.11-r2)
2025/02/04 22:46:58 INFO installing libpipeline (1.5.8-r0)
2025/02/04 22:46:58 INFO installing libbz2-1 (1.0.8-r10)
2025/02/04 22:46:58 INFO installing libxcrypt (4.4.38-r0)
2025/02/04 22:46:58 INFO installing libcrypt1 (2.40-r8)
2025/02/04 22:46:58 INFO installing perl (5.40.1-r0)
2025/02/04 22:46:58 INFO installing libstdc++ (14.2.0-r8)
2025/02/04 22:46:58 INFO installing groff (1.23.0-r3)
2025/02/04 22:46:58 INFO installing libseccomp (2.6.0-r0)
2025/02/04 22:46:58 INFO installing gdbm (1.24-r2)
2025/02/04 22:46:58 INFO installing man-db (2.13.0-r1)
2025/02/04 22:46:58 INFO installing ncurses-terminfo-base (6.5_p20241228-r0)
2025/02/04 22:46:58 INFO installing ncurses (6.5_p20241228-r0)
2025/02/04 22:46:58 INFO installing texinfo (7.2-r0)
2025/02/04 22:46:58 INFO installing trurl-doc (0.16-r1)
2025/02/04 22:46:58 INFO installing wolfi-keys (1-r8)
2025/02/04 22:46:58 INFO installing busybox (1.37.0-r0)
2025/02/04 22:46:58 INFO installing wolfi-base (1-r6)
2025/02/04 22:46:58 INFO OCI layer digest: sha256:6223686041cb5c90827ac98a92b6d07d92e1de4ebaf598629b145ba251d2c345
2025/02/04 22:46:58 INFO OCI layer diffID: sha256:265cb0af3cf8ff91315331312c5c2616e3cb6766f799880c8c52da8036c8de62
2025/02/04 22:46:58 INFO saving OCI image locally: apko.local/cache:331d6d073fb2ff75a5a056091ff5b19507159bc6c024c140a8e6bec5ae12ee1f
2025/02/04 22:47:00 WARN skipping local domain tagging apko.local/cache:331d6d073fb2ff75a5a056091ff5b19507159bc6c024c140a8e6bec5ae12ee1f as index.docker.io/library/melange:latest
2025/02/04 22:47:00 INFO running step "test/docs"
2025/02/04 22:47:00 WARN + '[' -d /home/build ]
2025/02/04 22:47:00 WARN + cd /home/build
2025/02/04 22:47:00 WARN + exit 0
2025/02/04 22:47:00 INFO running step "docs readability check"
2025/02/04 22:47:00 WARN + '[' -d /home/build ]
2025/02/04 22:47:00 WARN + cd /home/build
2025/02/04 22:47:00 WARN + basename /home/build/melange-out/trurl-doc
2025/02/04 22:47:00 WARN + doc_pkg=trurl-doc
2025/02/04 22:47:00 WARN + apk info -L trurl-doc
2025/02/04 22:47:00 WARN + wc -l
2025/02/04 22:47:00 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory
2025/02/04 22:47:00 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory
2025/02/04 22:47:00 WARN + '[' 3 -le 3 ]
2025/02/04 22:47:00 WARN + echo See:
2025/02/04 22:47:00 WARN + apk info -L trurl-doc
2025/02/04 22:47:00 INFO See:
2025/02/04 22:47:00 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory
2025/02/04 22:47:00 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory
2025/02/04 22:47:00 INFO trurl-doc-0.16-r1 contains:
2025/02/04 22:47:00 INFO var/lib/db/sbom/trurl-doc-0.16-r1.spdx.json
2025/02/04 22:47:00 INFO
2025/02/04 22:47:00 INFO This package [trurl-doc] is completely empty (i.e. installs no files).
2025/02/04 22:47:00 INFO Please check the package build for proper docs installation, and either:
2025/02/04 22:47:00 INFO   (a) fix the docs subpackage build to actually include documentation (check the split/manpages and split/infodir pipelines), or
2025/02/04 22:47:00 INFO   (b) remove the test/docs pipeline (if for some reason we want an empty docs package), or
2025/02/04 22:47:00 INFO   (c) remove the docs subpackage entirely
2025/02/04 22:47:00 WARN + echo 'This package [trurl-doc] is completely empty (i.e. installs no files).'
2025/02/04 22:47:00 WARN + echo 'Please check the package build for proper docs installation, and either:'
2025/02/04 22:47:00 WARN + echo '  (a) fix the docs subpackage build to actually include documentation (check the split/manpages and split/infodir pipelines), or'
2025/02/04 22:47:00 WARN + echo '  (b) remove the test/docs pipeline (if for some reason we want an empty docs package), or'
2025/02/04 22:47:00 WARN + echo '  (c) remove the docs subpackage entirely'
2025/02/04 22:47:00 WARN + exit 1
2025/02/04 22:47:00 INFO pod a4ef5ef5023ffe48a1be5d3776a65237f5905cb4e5dc3270e6b807bc54db2685 terminated
2025/02/04 22:47:00 INFO pod c1783aa04ab639e1663b6472e7e1d9b4c8b773f9821acf256609bb7c481ed164 terminated
2025/02/04 22:47:00 INFO ERROR: failed to test package. the test environment has been preserved:
2025/02/04 22:47:00 INFO   workspace dir: /home/user/tmp/melange-workspace-3876831066
2025/02/04 22:47:00 INFO   guest dir: /home/user/tmp/melange-guest-2721097524
2025/02/04 22:47:00 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:121: test/trurl] Error 1
```

--- a/curl.yaml
+++ b/curl.yaml
@@ -1,7 +1,7 @@
 package:
   name: curl
   version: "8.12.0"
-  epoch: 0
+  epoch: 1
   description: "URL retrieval utility and library"
   copyright:
     - license: MIT
@@ -60,6 +60,10 @@ subpackages:
     description: "headers for libcurl"
     pipeline:
       - uses: split/dev
+      # cd2nroff is needed by trurl
+      - runs: |
+          mkdir -p "${{targets.contextdir}}"/usr/bin
+          mv scripts/cd2nroff "${{targets.contextdir}}"/usr/bin
     dependencies:
       runtime:
         - brotli-dev

--- a/trurl.yaml
+++ b/trurl.yaml
@@ -1,7 +1,7 @@
 package:
   name: trurl
   version: "0.16"
-  epoch: 1
+  epoch: 2
   description: 'trurl is a command line tool for URL parsing and manipulation.'
   copyright:
     - license: curl
@@ -12,6 +12,7 @@ environment:
       - build-base
       - busybox
       - curl-dev
+      - perl
 
 pipeline:
   - uses: git-checkout
@@ -26,11 +27,18 @@ pipeline:
 
   - uses: strip
 
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/usr/share/man/man1
+      cd2nroff trurl.md > "${{targets.destdir}}"/usr/share/man/man1/trurl.1
+
 subpackages:
   - name: "trurl-doc"
     description: "documentation for trurl"
     pipeline:
       - uses: split/manpages
+    test:
+      pipeline:
+        - uses: test/docs
 
 update:
   enabled: true

